containers/ecr-viewer/src/app/services/formatService.tsx

```diff
@@ -592,6 +592,18 @@ export function toSentenceCase(str: string | undefined) {
   if (!str) return str;
   return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
 }
+/**
+ * Converts a string to title case, making the first character of each word uppercase.
+ * @param str - The string to convert to title case.
+ * @returns The converted title-case string. If the input is empty or not a string, the original input is returned.
+ */
+export function toTitleCase(str: string | undefined) {
```

Comment:
```suggestion
export const toTitleCase = (str: string | undefined) => {
```
just for more consistency with the other functions I see in the file's diff